src/frontend-samples/tooltip-customize-sample/TooltipCustomizeApp.tsx

```diff
@@ -2,97 +2,48 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-
-import "common/samples-common.scss";
-import { ProxyToolAdmin } from "api/showcasetooladmin";
-import { ElemProperty, TooltipCustomizeSettings } from "./TooltipCustomizeUI";
-import { HitDetail, imageElementFromUrl } from "@bentley/imodeljs-frontend";
-
-// SampleToolAdmin would typically extend ToolAdmin
-//  See Notes on use of ProxyToolAdmin at the bottom of this file.
-//  Do this: "class YourToolAdmin extends ToolAdmin"
-export class SampleToolAdmin extends ProxyToolAdmin {
-  public settings: TooltipCustomizeSettings = {
-    showImage: true,
-    showCustomText: false,
-    showElementProperty: true,
-    showDefaultToolTip: true,
-    customText: "Sample custom string",
-    elemProperty: ElemProperty.Origin,
+import React from "react";
```

Comment:
Wrong link in readme (Specify the content of a tooltip by implementing ToolAdmin.getToolTip.)

Reply:
Fixed.